### ISSUE TITLE
webpack `resolve` option doesn't work

### ISSUE THREAD

**Issue by mqklin** [ai/size-limit#71]
Is `resolve` option supposed to work? I can create a reproduce if it's necessary.  

 ```
ERROR  Size Limit can't resolve
App/dumb
in src/Root/App/Content
```
My package-json config: 
```
...
   "size-limit": [
     {
       "path": "src/index.js",
       "config": "webpack-config/app.js"
     }
  ],
...
```

My webpack config: 
```
...
  resolve: {
    alias: {
      App: path.resolve(root, 'src', 'Root', 'App'),
    },
  },
...
```

**Comment by ai** [comment]
Seems like it is happening because Size Limit overwrites `config.resolve`. Do you want to save your name in the project’s history and send PR? (we just need to set `config.resolve.modules`, so we can keep `config.resolve.alias`.

**Comment by ai** [comment]
Fixed 1736ab9
